src/error.rs

```diff
@@ -59,6 +59,9 @@ pub enum AppError {
 
     #[error("Unhandled Jwt error: {0}")]
     Jwt(#[from] jwt_simple::Error),
+
+    #[error("Bio exceeds maximum length")]
+    BioTooLong,
```

Comment:
```suggestion
    StringTooLong,
```
we use two different names, one `bio` and one `description`. So using a general name would make more sense if we are going with one name

Reply:
ofc, change the error message too